### PR TITLE
Release 6.5.0

### PR DESCRIPTION
**Pull request by TheMeier**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,27 @@ All notable changes to this project will be documented in this file.
...
+**Implemented enhancements:**
+
+- Add possibiliry to setup limits for user sessions [\#417](https://github.com/voxpupuli/puppet-systemd/issues/417)
+- add a cron-like systemd::timer interface [\#374](https://github.com/voxpupuli/puppet-systemd/issues/374)
+- Support `[Slice]` in manage\_unit and manage\_dropin [\#420](https://github.com/voxpupuli/puppet-systemd/pull/420) ([traylenator](https://github.com/traylenator))
+- feat: add a cron like wrapper for timers [\#419](https://github.com/voxpupuli/puppet-systemd/pull/419) ([TheMeier](https://github.com/TheMeier))
+
+**Closed issues:**
+
+- Running the example code for `systemd::manage_unit` from REFERENCE.md causes an error:  [\#414](https://github.com/voxpupuli/puppet-systemd/issues/414)
```
Comment (bastelfreak) [inline CHANGELOG.md:20]: I addes the `skip-changelog` label here because we already have the related PR in the changelog.

**Review by traylenator — APPROVED** [review]
Was thinking the same thing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,23 @@ All notable changes to this project will be documented in this file.
...
 These should not affect the functionality of the module.
 
-## [v6.4.0](https://github.com/voxpupuli/puppet-systemd/tree/v6.4.0) (2024-02-24)
+## [v6.5.0](https://github.com/voxpupuli/puppet-systemd/tree/v6.5.0) (2024-03-05)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-systemd/compare/v6.4.0...v6.5.0)
+
+**Implemented enhancements:**
+
+- Add possibiliry to setup limits for user sessions [\#417](https://github.com/voxpupuli/puppet-systemd/issues/417)
```
Comment (ekohl) [inline CHANGELOG.md:13]: I fixed a typo here, but in general I add `skip-changelog` when there is a PR that implements it. I'll leave it up to you whether you fix the typo, or skip it.

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by traylenator** [comment]
#422 is probably good for this - turned out to be much quicker to do than I expected.

File: CHANGELOG.md (hunk 2 above)
Comment (TheMeier) [reply CHANGELOG.md]: updated, thanks

**Comment by traylenator** [comment]
Sorry There's a mistake for CPUQuota - percentages bigger than 100% are okay - fixing...

**Comment by traylenator** [comment]
> Sorry There's a mistake for CPUQuota - percentages bigger than 100% are okay - fixing...

#423

**Comment by traylenator** [comment]
Don't wait for a fix for #424 - it's been broken a while and nobody can be using those parameters.

**Pull request merged**